providers/os/provider/provider.go

```diff
@@ -206,14 +206,6 @@ func (s *Service) Connect(req *plugin.ConnectReq, callback plugin.ProviderCallba
 		}
 	}
 
-	if inv == nil {
```

Comment:
I remember I added this code here. I also added a comment why it ended up being like this https://github.com/mondoohq/cnquery/pull/1617/files#r1311634791

Reply:
just tested this and it seems it's no longer an issue